### PR TITLE
Allow officers to enter a site description

### PR DESCRIPTION
**Pull request by benbaumann95**
### Description of change

- This follows the same schema as AssessmentDetail
- Do not inherit from PlanningApplicationPresenter but include a shared concern instead for reusable methods

### Story Link

https://trello.com/c/bJDX0XJi/1151-entering-site-description

### CODE REVIEW

File: app/presenters/concerns/assessment_tasks/assess_against_legislation_presenter.rb [hunk 1]
```diff
@@ -3,14 +3,15 @@
 module AssessmentTasks
   extend ActiveSupport::Concern
 
-  class AssessAgainstLegislationPresenter < PlanningApplicationPresenter
+  class AssessAgainstLegislationPresenter
```
Comment (benbaumann95) [inline app/presenters/concerns/assessment_tasks/assess_against_legislation_presenter.rb:6]: Still need to do some work on the validation request presenters not to inherit from `PlanningApplicationPresenter`

File: app/presenters/concerns/assessment_tasks/assessment_detail_presenter.rb [hunk 2]
```diff
@@ -3,10 +3,12 @@
 module AssessmentTasks
   extend ActiveSupport::Concern
 
-  class AssessmentDetailPresenter < PlanningApplicationPresenter
-    def initialize(template, planning_application, category)
-      super(template, planning_application)
+  class AssessmentDetailPresenter
+    include Presentable
```
Comment (allbecauseyoutoldmeso) [inline app/presenters/concerns/assessment_tasks/assessment_detail_presenter.rb:7]: Loving it 🎉 🎉 🎉

File: app/presenters/concerns/presentable.rb [hunk 3]
```diff
@@ -4,6 +4,9 @@ module Presentable
   extend ActiveSupport::Concern
   include Rails.application.routes.url_helpers
 
+  attr_reader :template, :planning_application
+
+  delegate :tag, :concat, :link_to, :truncate, :link_to_if, to: :template
```
Comment (allbecauseyoutoldmeso) [inline app/presenters/concerns/presentable.rb:9]: 🙌

**Review by allbecauseyoutoldmeso — APPROVED** [review]
Love how easy it is to add this because you've already put the framework in place 🙌 

Just wondering if, even if it's repetitive, it's worth adding a system spec?

**Comment by benbaumann95** [comment]
> Love how easy it is to add this because you've already put the framework in place 🙌
> 
> Just wondering if, even if it's repetitive, it's worth adding a system spec?

Thanks for the reminder! It's a change locally i didn't commit by mistake so pushing now

**Pull request merged**